dash/dash.py

```diff
@@ -727,6 +727,10 @@ def simple_clone(c, children=None):
                 return cls(**props)
 
             layout_value = self._layout_value()
+
+            if isinstance(layout_value, (list, tuple)):
+                layout_value = html.Div(children=layout_value)
```

Comment:
We can  delete the function [`simple_clone`](https://github.com/plotly/dash/blob/bbd013cd9d97d8c41700b240dd95c32e6b875998/dash/dash.py#L711-L727) and assign the `self.validation_layout = layout_value`  instead.

Reply:
thanks, will make requested changes

Reply:
done